styles/style.css

```diff
@@ -0,0 +1,148 @@
+*{
+    margin:0;
+    padding:0;
+}
+html{
+    font-size: 62.5%;
+}
+body{
+    font-size: 1.6rem;
+}
+ul{
+    list-style: none;
+}
+.header{
+    padding : 0% 10%;
+}
+.header img{
+    max-height: 70px;
+}
+.banner{
+    padding: 40px 10%;
+    align-items: center;
+    display: flex;
+    background-color: #171A29;
+}
+.banner img{
+    max-height: 200px;
+}
+h1{
+    margin-left: 70px;
+    font-weight: normal;
+    color: #FFFFFF;
+    font-size: 3rem;
+}
+main{
+    min-height: calc(100vh - 155px);
+}
+.content{
+    display: flex;
+}
+.categories{
+    padding: 30px 30px 0 0;
+    text-align: right;
+    border-right: 1.5px #F6F6F7 solid;
+}
+
+.categories ul li{
+    margin: 10px 0;
+}
+.recommended{
+    font-size: 1.7rem;
+    font-weight:600;
+    color: #ffa500;
+}
+.menu{
+    padding: 30px 30px;
+}
+h2{
+    font-size: 3rem;
+    color: #171A29;
+}
+.menu > p ,.cart > p{
+    font-size: 1.3rem;
+    color: #808080;
+}
+.dish img{
+    height: 12px;
+}
+.dish{
+    padding: 25px 0;
+    border-bottom: 1.5px #F6F6F7 solid;
+}
+.price{
+    font-size: 1.3rem;
+    color: #36454F;
+}
+.dish-name{
+    font-size: 1.7rem;
+    margin: 15px 0;
+    color: #36454F;
+}
+.cart{
+    padding: 30px;
+    padding-right: 175px;
+}
+.cart ul{
+    margin: 30px 0;
+}
+.amount{
+    display: flex;
+    justify-content: space-between;
+}
+h3{
+    font-size: 1.7rem;
+}
+.disclaimer{
+    font-size: 1.4rem;
+    font-weight: 100;
+    color: #808080;
+    margin-bottom: 40px;
+}
+.amount p{
+    color: black;
+    font-size: 1.7rem;
+    font-weight: bold;
+}
+.check-out-button{
+    cursor: pointer;
+    font-size: 1.4rem;
+    font-weight: bold;
+    border: none;
+    width: 100%;
+    color: #FFFFFF;
+    text-align: center;
+    padding:10px 0;
+    background-color: #32CD30;
+}
+footer{
+    padding:5px 0;
+    background-color: black;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    text-align: center;
+}
+footer img{
+    max-width:140px;
```

Comment:
This is fine for now, but going forward, as discussed, give the dimensions in `rem`